zds/utils/templatetags/remove_url_scheme.py

```diff
@@ -5,8 +5,8 @@
 register = template.Library()
 
 
-@register.filter('remove_url_protocole')
-def remove_url_protocole(input_url):
+@register.filter('remove_url_scheme')
+def remove_url_scheme(input_url):
```

Comment:
Ouais. Tu peux littéralement inverser les args de `in` et remplacer `in` par `.startswith`, done.